doc/man5/config.pod

```diff
@@ -378,7 +378,7 @@ For example:
 
 =head1 EXAMPLES
 
-This example shows how to using quoting and escaping.
+This example shows how use using quoting and escaping.
```

Comment:
This does not look right even after the edit. :smile:

Reply:
:) indeed.